HotTips/Tips/EDI_FormatDoc.md

```diff
@@ -0,0 +1 @@
+﻿Press **Ctrl+K+D** to format the current document to code formatting settings. No need to use space/delete to format curly braces and indentation.
```

Comment:
And what about Ctrl+K, F to format the current selection.
(These two should be on the same tip.)